xwiki-platform-core/xwiki-platform-blog/xwiki-platform-blog-ui/src/main/resources/Blog/BlogStyle.xml

```diff
@@ -260,15 +261,6 @@
 .entry-author-label {
   display: inline;
 }
-/* Pseudo-tools visible in the edit mode */
-div.publish-message {
-  background-image: url("$xwiki.getSkinFile('icons/silk/page_white_world.png')");
-  background-position: 6px center;
-}
-div.hide-message {
-  background-image: url("$xwiki.getSkinFile('icons/silk/lock_open.png')");
-  background-position: 6px center;
-}
```

Comment:
Answering my question: I think the paradigm has changed and an icon is no longer represented by a URL. Which is why the icon SS only offers render*() methods.